includes/Withdraw/functions.php

```diff
@@ -107,13 +107,29 @@ function dokan_withdraw_get_method( $method_key ) {
 function dokan_withdraw_get_method_title( $method_key ) {
     $registered = dokan_withdraw_register_methods();
 
-    if ( isset( $registered[ $method_key ] ) ) {
+    if ( 'custom' === $method_key ) {
+        return dokan_get_option( 'withdraw_method_name', 'dokan_withdraw' );
+    } elseif ( isset( $registered[ $method_key ] ) ) {
         return $registered[ $method_key ]['title'];
     }
 
     return '';
 }
 
+/**
+ * Get withdraw method details (email/phone)
+ *
+ * @param object $request
+ *
+ * @return string
+ */
+function render_withdraw_method_details( $request ) {
```

Comment:
changed this with `dokan_render_custom_withdraw_method_details` also change method return type to `void|string`

Reply:
remove this method and change function dokan_withdraw_get_method_title() with below  code

```
/**
 * Get title from a withdraw method
 *
 * @param string $method_key
 * @param object|null $request
 *
 * @return string
 */
function dokan_withdraw_get_method_title( $method_key, $request = null ) {
    $registered = dokan_withdraw_register_methods();

    if ( ! isset( $registered[ $method_key ] ) ) {
        return '';
    }

    $title = $registered[ $method_key ]['title'];

    if ( null !== $request ) {
        $details = maybe_unserialize( $request->details );
        if ( isset( $details['value'] ) && 'custom' === $request->method ) {
            $title .= '- ' . $details['value'];
        }
    }

    return $title;
}
```